lib/widgets/header2/widgets/chunks/attributes/levels.dart

```diff
@@ -19,14 +19,14 @@ class LevelFlagsDisplay extends StatelessWidget {
           final attributes = chunkAttributes as RessAttributes;
           return [
             _LevelRow(
-              title: "res",
+              title: "res:",
               levels: [
-                _Level(label: "1", isOn: attributes.isRes1),
-                _Level(label: "2", isOn: attributes.isRes2),
-                _Level(label: "3", isOn: attributes.isRes3),
-                _Level(label: "4", isOn: attributes.isRes4),
-                _Level(label: "5", isOn: attributes.isRes5),
-                _Level(label: "6", isOn: attributes.isRes6),
+                _Level(label: " 1 ", isOn: attributes.isRes1),
```

Comment:
you should have used a padding in the widget to create the spacing. I will fix that